src/utils/context.tsx

```diff
@@ -83,6 +89,17 @@ const ContextProvider = ({ children }: React.PropsWithChildren<{}>) => {
     return yyyy + "-" + mm + "-" + dd
   }
 
+  const { loading, error, data } = useQuery(QUERY_DAILY_ENTRIES)
+
+  useEffect(() => {
```

Comment:
How often does this run? I'm assuming having it in our Context is what allows our app state to be in sync with the data. Nice solution!